common/djangoapps/entitlements/tasks.py

```diff
@@ -19,7 +19,12 @@
 MAX_RETRIES = 11
 
 
-@task(bind=True, ignore_result=True, routing_key=ROUTING_KEY)
+@task(
+    bind=True,
+    ignore_result=True,
+    routing_key=ROUTING_KEY,
+    name='entitlements.expire_old_entitlements',
```

Comment:
To avoid the rename, this should have been `entitlements.tasks.expire_old_entitlements`.